Project.toml

```diff
@@ -13,27 +13,29 @@ Preferences = "21216c6a-2e73-6563-6e65-726566657250"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 SpecialFunctions = "276daf66-3868-5448-9aa4-cd146d93841b"
-StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+StaticArraysCore = "1e83bf80-4336-4d27-bf5d-d5a4f845583c"
 
 [compat]
 Calculus = "0.2, 0.3, 0.4, 0.5"
 CommonSubexpressions = "0.3"
-DiffResults = "0.0.1, 0.0.2, 0.0.3, 0.0.4, 1.0.1"
+DiffResults = "1.1.0"
 DiffRules = "1.4.0"
 DiffTests = "0.0.1, 0.1"
 LogExpFunctions = "0.3"
 NaNMath = "0.2.2, 0.3, 1"
 Preferences = "1"
 SpecialFunctions = "0.8, 0.9, 0.10, 1.0, 2"
-StaticArrays = "0.8.3, 0.9, 0.10, 0.11, 0.12, 1.0"
-julia = "1"
+StaticArrays = "1.5.7"
```

Comment:
```suggestion
```

Reply:
I guess that's fine, loading StaticArraysCore implies this bound during testing?

Reply:
I don't think it is allowed to have compat for non-dependencies. If this is needed you can keep the StaticArrays dependency (without `using` it).